### PR TITLE
Update 1-Installer-le-projet-en-local.md

### PR DESCRIPTION
**Pull request by mstrom**
Sorry I don't speak French but I'd like to propose this change so any developer who use Windows can easyly contribute to project by using Windows port of make. I tried to run Linus command line under Win 10 but its not working (because it tried to run docker under linux)

Probably I need help to translate this into French

### CODE REVIEW

**Comment by tgalopin** [comment]
Hello @mstrom !

Thanks for your proposal, I will translate it in french when I find the time. It's awesome you took time to create a PR on a project in a language you don't speak!

**Comment by Stoakes** [comment]
Hello, here is the french translation of your text :

**Note**: Pour les utilisateur de Windows, vous pouvez utiliser [l'équivalent Windows de make](http://gnuwin32.sourceforge.net/packages/make.htm)  

```
**Note**: Pour les utilisateur de Windows, vous pouvez utiliser [l'équivalent Windows de make](http://gnuwin32.sourceforge.net/packages/make.htm)  
```

Cheers

**Comment by tgalopin** [comment]
Thanks a lot @Stoakes! @mstrom I think you are able to finish this now :) .

**Comment by mstrom** [comment]
@Stoakes thank you!

@tgalopin Translation committed

File: docs/1-Installer-le-projet-en-local.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ Il ne comporte AUCUNE GARANTIE, dans la mesure de ce que permet la loi.
 > **Note**: si vous utilisez Windows, nous vous recommandons très fortement d'utiliser la console Linux intgrée à
 > Windows 10 (https://msdn.microsoft.com/fr-fr/commandline/wsl/install_guide) ou d'utiliser un émulateur de ligne de
 > commande pour pouvoir utiliser `make` qui vous facilitera grandement le travail.
+**Note**: Pour les utilisateur de Windows, vous pouvez utiliser [l'équivalent Windows de make](http://gnuwin32.sourceforge.net/packages/make.htm)
```
Comment (tgalopin) [inline docs/1-Installer-le-projet-en-local.md]: There is a small mistake: `utilisateurs` instead of `utilisateur`.

Comment (mstrom) [inline docs/1-Installer-le-projet-en-local.md]: Changed

**Comment by tgalopin** [comment]
Thanks @mstrom !

**Pull request merged**